client/src/assets/scss/main.scss

```diff
@@ -1,12 +1,17 @@
 @import "./foundation/settings";
 @import "~foundation-sites/scss/foundation";
-@import "./foundation/variables.scss";
+@import "./variables.scss";
 @include foundation-everything;
 
 @import "./app.scss";
 @import "./topbar.scss";
 @import "./trails.scss";
 
+.button-group {
+  border-radius: 20px;
+  background: #f17b30;
+}
```

Comment:
Looks like this class `button-group` also exists in `topbar`

I would recommend consistency in colors (either color variables like you started, or using hex codes like you are here)